src/main/java/com/jland/website/web/controller/ParticipantController.java

```diff
@@ -28,9 +34,12 @@ public String notParticipate(@PathVariable("userId") long userId, @PathVariable(
     }
 
     @GetMapping(path = "/participants")
-    public String getAll(Model model, @PathVariable("conferenceId")  long conferenceId){
-        model.addAttribute("participants", participantService.getAll(conferenceId));
-
+    public String getAll(Model model, @PathVariable("conferenceId")  long conferenceId, Principal principal){
+        List<Participant> participants = participantService.getAll(conferenceId);
+        model.addAttribute("participants", participants);
+        model.addAttribute("conferenceId", conferenceId);
+        model.addAttribute("user", userService.findUserByUsername(principal.getName()));
+        model.addAttribute("count", participants.size());
        return "participants";
```

Comment:
why you don't return result in ResponseEntity?

Reply:
there is not special adjustment for status and header in current controllers